### DISCUSSION TITLE
SONOFF POW R3 Zero Values

### DISCUSSION THREAD

**Discussion by reset12** [arendst/Tasmota#16331] [Q&A] [answered]
Hello,
i have flashed SONOFF POW R3 with tasmota 12.1.0 and configured SONOFF POW R2 because POW R3 is not available.
I got always all values Zero in Display no difference when i switch the Lamp on/off.

I have tried to calibrate like the manual, but nothing changes.

![Screenshot 2022-08-24 at 16-24-14 Tasmota - Main Menu](https://user-images.githubusercontent.com/14790813/186444220-746d619b-5444-45b2-96cd-6402cf26098d.png)
![Screenshot 2022-08-24 at 16-27-09 Tasmota - Configure Module](https://user-images.githubusercontent.com/14790813/186444489-3feda03a-c363-46f2-9fe2-31654f3593f6.png)

**Comment by sfromis** [comment, marked as answer]
Yeah bad config when your device is not a Sonoff Pow R2. You need to apply the template from the repository.
https://templates.blakadder.com/sonoff_POWR3.html

**Comment by reset12** [comment]
Thank you, 
I works now great.
